ci/check-signed-off-by.sh

```diff
@@ -32,6 +48,10 @@ for sha in $commits; do
   author="Signed-off-by: $(git show -s --format="%an <%ae>" ${sha})"
   committer="Signed-off-by: $(git show -s --format="%cn <%ce>" ${sha})"
 
+  if commit_by_dependabot "${sha}"; then
+    continue
+  fi
+
```

Comment:
Instead of creating a function and re-doing the git work why not just check for the expected text appended in the already existing variables?

I would suggest something on the lines of:

```sh
# XXX: somewhere at the beginning
BOT_AUTHOR="dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>"
BOT_COMMITER="GitHub <noreply@github.com>

# XXX: The skip dependabot test
if [[ $author == *$BOT_AUTHOR && $committer == *$BOT_COMMITTER ]]; then
  continue
fi
```

```

Reply:
Removed the function.